crates/core/tedge_watchdog/src/systemd_watchdog.rs

```diff
@@ -242,27 +251,55 @@ mod tests {
         let (mut sender, mut receiver) = mpsc::unbounded::<Message>();
         let health_topic = Topic::new("tedge/health/test-service").expect("Valid topic");
 
-        for x in 1..5i64 {
+        for _x in 1..5i64 {
             let health_status = json!({
                 "status": "up",
                 "pid": 123u32,
-                "time": x,
+                "time": "2013-06-22T17:03:14.000+02:00",
```

Comment:
The point of the loop was to have a set of health status messages with increasing timestamp values, not a static value like this. So, may be you can generate a few that increases by a second or so.

Reply:
fixed